examples/cityscapes/README.md

```diff
@@ -0,0 +1,182 @@
+
+**Prerequisites**
+
+Before you begin, ensure that you have the following prerequisites installed:
+
+- **[Python](https://www.python.org/)** (version 3.8 or higher)
+- **[Poetry](https://python-poetry.org/)**
+
+### Tensorleap **CLI Installation**
+
+with `curl`:
+
+```
+curl -s <https://raw.githubusercontent.com/tensorleap/leap-cli/master/install.sh> | bash
+```
+
+with `wget`:
+
+```
+wget -q -O - <https://raw.githubusercontent.com/tensorleap/leap-cli/master/install.sh> | bash
+```
+
+CLI repository: https://github.com/tensorleap/leap-cli
+
+### Tensorleap CLI Usage
+
+#### Tensorleap **Login**
+To allow connection to your Tensorleap platform via CLI you will have to authenticate and login.
+To login to Tensorealp:
+
+```
+leap auth login [api key] [api url].
+```
+
+- API Key is your Tensorleap token (see how to generate a CLI token in the section below).
+- API URL is your Tensorleap environment URL: CLIENT_NAME.tensorleap.ai
+
+<br> 
+
+**How To Generate CLI Token from the UI**
+
+1. Login to the platform in 'CLIENT_NAME.tensorleap.ai'
+2. Scroll down to the bottom of the **Resources Management** page, then click `GENERATE CLI TOKEN`  in the bottom-left corner.
+3. Once a CLI token is generated, just copy the whole text and paste it into your shell:
+
+```
+leap auth login [api key] [api url]
+```
+
+### Tensorleap Dataset Deployment
+
+To deploy your local changes:
+
+```
+leap code push
+```
+
+### **Tensorleap files**
+
+Tensorleap files in the repository include `leap_binder.py` and `leap.yaml`. The files consist of the  required
+configurations to make the code integrate with the Tensorleap engine:
+
+### **leap.yaml file**
+leap.yaml file is configured to a dataset in your Tensorleap environment and is synced to the dataset saved in the environment.
+
+For any additional file being used we add its path under `include` parameter:
+
+```
+include:
+    - leap_binder.py
+    - utils_all/gcs_utils.py
+    - utils_all/general_utils.py
+    - utils_all/metrics.py
+    - utils_all/preprocessing.py
+    - visualizers/visualizers.py
+    - yolo_helpers/yolo_utils.py
+    - project_config.py
+```
+
+### **leap_binder.py file**
+`leap_binder.py` configure all binding functions used to bind to Tensorleap engine. These are the functions used to 
+evaluate and train the model, visualize the variables, and enrich the analysis with external metadata variables
+
+### Testing
+
+To test the system we can run `leap_test.py` file using poetry:
+
+```
+poetry run test
+```
+
+This file will execute several tests on [the leap_binder.py] script to assert that the implemented binding functions: 
+preprocess, encoders,  metadata, etc,  run smoothly.
+
+*For further explanation please refer to the [docs](https://docs.tensorleap.ai/)*
+
+## Project Description
+_____________________________________________
+### YOLOv7 model with Cityscapes dataset
+
+This project utilizes the [YOLOv7]() algorithm to perform object detection on [Cityscapes]() dataset. The 
```

Comment:
Not sure we need to introduce cityscapes. Maybe just in two words saying It is one of the standard ADAS benchmarks used for multiple vision related tasks.